plugin/pkg/scheduler/schedulercache/cache.go

```diff
@@ -104,7 +104,11 @@ func (cache *schedulerCache) List(selector labels.Selector) ([]*v1.Pod, error) {
 func (cache *schedulerCache) FilteredList(podFilter PodFilter, selector labels.Selector) ([]*v1.Pod, error) {
 	cache.mu.Lock()
 	defer cache.mu.Unlock()
-	var pods []*v1.Pod
+	maxSize := 0
```

Comment:
Please add a comment explaining that in the current code, the returned list always has almost as many entries as the total number of pods and that's why preallocating the array by total number of pods makes sense.

Reply:
Done